assets/pages/new-contributions-landing/contributionsLandingActions.js

```diff
@@ -127,6 +132,47 @@ const setThirdPartyPaymentLibrary =
 
 const setPayPalHasLoaded = (): Action => ({ type: 'SET_PAYPAL_HAS_LOADED' });
 
+const setisSignInRequired = (isSignInRequired: boolean): Action => ({
+  type: 'SET_SIGN_IN_REQUIRED',
+  isSignInRequired,
+});
+
+const setIdentityRequestPending = (isIdentityRequestPending: boolean): Action => ({
+  type: 'SET_IDENTITY_REQUEST_PENDING',
+  isIdentityRequestPending,
+});
+
+const setLastIdentityResponse = (lastIdentityResponse: IdentityResponse): Action => ({
+  type: 'SET_LAST_IDENTITY_RESPONSE',
+  lastIdentityResponse,
+});
+
+const checkIfEmailHasPassword = (email: string) =>
+  (dispatch: Dispatch<Action>, getState: () => State): void => {
+    const state = getState();
+    dispatch(setIdentityRequestPending(true));
+    fetchJson(
+      `/dummy/${encodeURIComponent(email)}`,
```

Comment:
change this to ``${routes.getuser-type}?email=${encodeURIComponent(email)}`` to make it work with the new endpoint